### DISCUSSION TITLE
SMB Support?

### DISCUSSION THREAD

**Discussion by markfm62** [hardcore-sushi/DroidFS#36] [Q&A]
Any chance of adding hooks to allow mounting an offboard volume on a LAN share via SMB?  The scenario is a NAS storing encrypted volumes.

I took a look in the Unsafe Features section, nothing stood out.  I tried the "Allow opening files..." Option, but Solid Explorer didn't list DroidFS as an option to open a file.

**Comment by hardcore-sushi** [comment]
Please see: #25.

Th unsafe feature "Allow opening files with other applications" only allows files inside encrypted volumes to be opened with other apps.

DroidFS can't open files shared by other apps, only import them inside encrypted volumes.

**Reply by markfm62** [reply]
Thanks for the rapid response!
